lib/smart_answer/calculators/benefit_cap_calculator_data_query.rb

```diff
@@ -1,8 +1,28 @@
 module SmartAnswer::Calculators
   class BenefitCapCalculatorDataQuery
-    attr_reader :data
+    attr_reader :data, :benefits, :questions
```

Comment:
This is unnecessary as you've also defined `benefits` and `questions` methods below.